index.ts

```
@@ -48,18 +36,21 @@ export async function run(args: { [k: string]: boolean | string }): Promise<void
     usage(""); // no return
   } else {
     const signer = await retrieveSignerFromCLIArgs();
-    do {
-      try {
-        // One global signer for use with a specific per-chain provider.
-        // todo: Support a void signer for monitor mode (only) if no wallet was supplied.
-        await cmds[cmd](logger, signer);
-      } catch (error) {
-        if (await processCrash(logger, cmd, config.pollingDelay, error as AnyObject)) {
-          // eslint-disable-next-line no-process-exit
-          process.exit(1);
-        }
-      }
-    } while (config.pollingDelay !== 0);
+    try {
+      // One global signer for use with a specific per-chain provider.
+      // todo: Support a void signer for monitor mode (only) if no wallet was supplied.
+      await cmds[cmd](logger, signer);
+    } catch (error) {
+      logger.error({
+        at: "index",
+        message: "There was an execution error!",
+        reason: error,
+        e: error,
+        error,
```

This was copy/pasted from the existing `processCrash()`. I'm not sure why we'd be reporting `error` as `reason` _and_ `e` :thinking:. iirc some part of the logging subsystem does tend to eat any error objects emitted directly in the log object, though.